modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/operation/construct/MuleOperation.java

```diff
@@ -61,22 +75,29 @@ public CompletableFuture<Event> execute(Event event) {
 
   @Override
   public void initialise() throws InitialisationException {
+    processingStrategy = getDefaultProcessingStrategyFactory(muleContext).create(muleContext, operationModel.getName());
```

Comment:
are there tasks to make this consistent with other places where chains are created?